src/CaptureClient/CaptureEventProcessorTest.cpp

```diff
@@ -1169,29 +1169,32 @@ TEST(CaptureEventProcessor, CanHandleThreadStateSlices) {
   event_processor->ProcessEvent(runnable_event);
   event_processor->ProcessEvent(dead_event);
 
+  ASSERT_TRUE(actual_running_thread_state_slice_info.has_value());
+  ASSERT_TRUE(actual_runnable_thread_state_slice_info.has_value());
+  ASSERT_TRUE(actual_dead_thread_state_slice_info.has_value());
```

Comment:
Personally I would have placed each of these right before the corresponding `EXPECT_EQ`s below.

Reply:
Done.